packages/graphql/src/translate/create-create-and-params.ts

```diff
@@ -154,7 +155,7 @@ function createCreateAndParams({
                         refNode,
                         context,
                     });
-                    res.creates.push(connectOrCreateQuery);
+                    res.creates.push(wrapInCall(connectOrCreateQuery, varName));
```

Comment:
@angrykoala, I decided to put the `wrapInCall` in this file rather than in https://github.com/neo4j/graphql/blob/dev/packages/graphql/src/translate/connect-or-create/create-connect-or-create-and-params.ts#L66 as it broke quite a few integration tests.

Tagging @darrellwarde and @danstarns for visibility

Reply:
@tbwiss After some quick checks, I think we can move `wrapInCall` inside `createConnectOrCreateAndParams`, but we need to remove a `wrapInCall` in `create-update-and-params`. I think is worth making this change to avoid code duplicity

Reply:
Will make the change!